pkg/cmd/admin/network/unisolate_project.go

```diff
@@ -0,0 +1,78 @@
+package network
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	kerrors "k8s.io/kubernetes/pkg/util/errors"
+
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const (
+	UnIsolateProjectNetworkCommandName = "unisolate-projects"
```

Comment:
I still don't like "unisolate" as the name for this operation, since I think it only implies "everyone can access this project", not "this project can access everyone". Something with "admin" in the name would be better IMHO